examples/kfp/config.yaml.example

```diff
@@ -17,6 +17,8 @@
 project_id: [PROJECT ID]
 bucket_id: [BUCKET ID]
 region: "us-central1"
+cluster_name: [GKE CLUSTER NAME]
+cluster_zone: [GKE CLUSTER ZONE]
```

Comment:
duplicate

Reply:
Those are two different params and both required to identify the cluster to apply WI for. Could you clarify further?

Reply:
oh oops, misread sorry